### PR TITLE
Rework raw block insertion

### PR DESCRIPTION
**Pull request by muratovv**
### Description of the Change

* rework Irohad initialization with storage block insertions
* rename raw block insertion to raw block loader
* remove insertion stuff from block loader
* fix integration tests
* remove redundant tests from block insertion tests

### Benefits

Easier to manage Irohad because of simplification of initialization.

### CODE REVIEW

File: irohad/main/impl/raw_block_loader.cpp [hunk 1]
```diff
@@ -38,16 +33,7 @@ namespace iroha {
...
-      auto storage = factory_->createMutableStorage();
-      for (auto &&block : blocks) {
-        storage->apply(block, [](const auto &current_block, auto &query,
-                                 const auto &top_hash) { return true; });
-      }
-      factory_->commit(std::move(storage));
-    }
-
-    nonstd::optional<std::string> BlockInserter::loadFile(std::string path) {
+    nonstd::optional<std::string> BlockLoader::loadFile(std::string path) {
```
Comment (kamilsa) [inline irohad/main/impl/raw_block_loader.cpp]: Is there a case when nullopt can be returned? There should be either check on that file exists or function should return just string. IMO first option is better

File: test/module/irohad/common/block_insertion_test.cpp [hunk 2]
```diff
@@ -17,81 +17,43 @@
...
+/**
+ * @given generated block
+ *
+ * @when convert block to JSON
+ * AND parseBlock() with BlockLoader API
+ *
+ * @then check that blocks are equal
+ */
+TEST(BlockLoaderTest, BlockLoaderWhenParseBlock) {
   cout << "----------| block => string_repr(block)"
```
Comment (kamilsa) [inline test/module/irohad/common/block_insertion_test.cpp]: Do we really need this cout here?

Comment (muratovv) [reply test/module/irohad/common/block_insertion_test.cpp]: No, I 'll fix it.

File: test/module/irohad/common/block_insertion_test.cpp [hunk 3]
```diff
@@ -17,81 +17,43 @@
 
 #include <gmock/gmock.h>
```
Comment (lebdron) [inline test/module/irohad/common/block_insertion_test.cpp]: gmock is not required in this test.

File: irohad/main/raw_block_loader.hpp [hunk 4]
```diff
@@ -35,9 +35,9 @@ namespace iroha {
      * This class will be useful for creating test environment
      * and testing pipeline.
      */
-    class BlockInserter {
+    class BlockLoader {
      public:
-      explicit BlockInserter(std::shared_ptr<ametsuchi::MutableFactory> factory);
+      BlockLoader() = default;
```
Comment (lebdron) [inline irohad/main/raw_block_loader.hpp]: Logger has to be initialized in constructor, `std::shared_ptr<ametsuchi::MutableFactory> factory_;` is not required now.

File: test/module/irohad/common/raw_block_loader_test.cpp [hunk 5]
```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright Soramitsu Co., Ltd. 2017 All Rights Reserved.
```
Comment (l4l) [inline test/module/irohad/common/raw_block_loader_test.cpp]: too old :(

**Pull request merged**